android-pay/src/main/java/com/stripe/wrap/pay/utils/CartManager.java

```diff
@@ -396,18 +397,57 @@ public Cart buildCart() throws CartContentException {
         }
     }
 
+    /**
+     * Get the current total price for the cart. If the value has been manually set or previously
+     * calculated, the cached value is returned. If no such value has been set (or the value
+     * has been invalidated by the addition or removal of items), then a new value is calculated,
+     * cached, and returned.
+     *
+     * @return the total price of the items in this CartManager, or {@code null} if that value
+     * is neither set nor able to be calculated
+     */
+    @Nullable
```

Comment:
It would be ridiculous not to allow users to check the total price of the cartmanager.